index.js

```diff
@@ -153,5 +186,34 @@ server.on('playerJoin', async function(client) {
 	connected = !client.ended;
 });
 
-Object.values(handlers).forEach((handler) => handler.obtainHandlers(handlers, connect));
-console.log('\x1b[33mMiniblox Translation Layer Started!\nDeveloped & maintained by 7GrandDad (https://youtube.com/c/7GrandDadVape)\nVersion: ' + VERSION + '\x1b[0m');
\ No newline at end of file
+Object.values(handlers).forEach((handler) =>
+	handler.obtainHandlers(handlers, connect)
+);
+
+(async () => {
+	let r = await fetch("https://miniblox.io/", {
+		method: "GET",
+		headers: {
+			"User-Agent":
+				"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/133.0.0.0 Safari/537.36 Edg/133.0.0.0",
+		},
+	});
+	let js = (await r.text()).match(/\/assets\/index-[a-zA-Z0-9]*\.js/g)[0];
+
+	r = await fetch("https://miniblox.io" + js, {
```

Comment:
```suggestion
	r = await fetch(`https://miniblox.io${js}`, {
```

NOOOOOO AAAAAAAAAAAAAAA MY EYES
he really did use `+` concatenation in JS